docs/en/deployment.md

```diff
@@ -1,58 +1,83 @@
 Before you launch the solution, review the architecture, supported regions, and other considerations discussed in this guide. Follow the step-by-step instructions in this section to configure and deploy the solution into your account.
 
-**Time to deploy**: Approximately 10 minutes
 
-## Deployment overview
+!!! notice
 
-Use the following steps to deploy this solution on AWS.
+    It's good practice to set a billing alarm before deploying this solution. Refer to this [link](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/monitor_estimated_charges_with_cloudwatch.html)
 
-* Step1: Check your QuickSight account
-* Step2: Launch the AWS CloudFormation template into your AWS account to deploy the solution.
-* Step3: Update QuickSight permissions
 
+**Time to deploy**: Approximately 10 minutes
 
-!!! notice
 
-    Step1 and Step3 are one time setup per account when deploying this solution for the first time. You don't have to do them when repeating deployment
+## Prerequisites
 
-## Deployment
+### Sign up for QuickSight
 
-### Step1: Check your QuickSight
+!!! notice
+    You need correct AWS Identity and Access Management (IAM) permissions to sign up for QuickSight, please refer to [QuickSight document](https://docs.aws.amazon.com/quicksight/latest/user/signing-up.html)
```

Comment:
it's better using the title of doc, `[Signing up for an Amazon QuickSight subscription](...)`

Reply:
changed, thanks.